### PR TITLE
Small fixes for rucio publisher

### PR DESCRIPTION
**Pull request by belforte**
this PR addresses a couple of bugs and a pylint complain found by pylint when running on PR https://github.com/dmwm/CRABServer/pull/8562

```text
Warnings from pylint by file and severity

    src/python/Publisher/Main.py
    src/python/Publisher/PublisherMasterRucio.py
        E1111, line 396 in Master.markAsFailed: Assigning result of a function call, where the function has no return
        E1123, line 396 in Master.markAsFailed: Unexpected keyword argument 'crabserver' in function call
        W1203, line 317 in Master.startSlave: Use lazy % formatting in logging functions
    src/python/Publisher/RunPublisher.py
    src/python/Publisher/TaskPublishRucio.py
        E1120, line 115 in publishInDBS3.publishOneBlockInDBS: No value for argument 'migLogDir' in function call
```

### CODE REVIEW

**Comment by belforte** [comment]
@novicecpp I did not squash commits because it may be easier to review one commit at a time

File: src/python/Publisher/PublisherMasterRucio.py [hunk 1]
```diff
@@ -313,7 +313,7 @@ def startSlave(self, task):  # pylint: disable=too-many-branches, too-many-local
                     lfnsToPublish.append(fileDict['destination_lfn'])
             FilesInfoFromTBDInBlock[blockName] = filesInfo
 
-        logger.info(f"Prepare publish info for {len(blocksToPublish)} blocks")
+        logger.info("Prepare publish info for %s blocks", len(blocksToPublish))
```
Comment (novicecpp) [inline src/python/Publisher/PublisherMasterRucio.py:316]: Did we agree sometimes ago that we will use f-string in log format?
https://stackoverflow.com/a/59027338

We (sorry for blaming everybody here) just too lazy to change pylint config.

**Review by novicecpp — APPROVED** [review]
Thanks for fixing Stefano.

Before merging this PR, could you please rebase this PR to master once to let pylint run again? I have merged my PR https://github.com/dmwm/CRABServer/pull/8562 minutes ago.

File: src/python/Publisher/PublisherMasterRucio.py (hunk 1 above)
Comment (belforte) [reply src/python/Publisher/PublisherMasterRucio.py:316]: I would very much like to use f-strings in logging calls, but do not know how to change pylint. Maybe we need Alan ? @mapellidario

**Comment by belforte** [comment]
rebased, maybe there's a way to do w/o the force push... sorry for being clumsy

This spotted a typo in argument name (`crabserver` vs `crabServer`). THANKS THANK THANKS for suggestin to do this check

**Comment by novicecpp** [comment]
> rebased, maybe there's a way to do w/o the force push... sorry for being clumsy

No, rebase PR always ends up force-push.

**Pull request merged**